### DISCUSSION TITLE
Adxe project created  but i did not get simulator option in it.

### DISCUSSION THREAD

**Discussion by ghost** [axmolengine/axmol#630] [Q&A] [answered]
hello the project successfully created but don't able to get simulator option in it.. when i gone through the terminal section i was found following things failed.. what i have to do.. i am using m1 mac with big sur 11.3.1 and i am building for ios simulator for on a Mac OS X.  

Detecting C compiler ABI info
-- Detecting C compiler ABI info - failed
-- Check for working C compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang
-- Check for working C compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang - works 

i am using this cmake command to create ios simulaor build..

cmake -S . -B simbuild -GXcode -DCMAKE_TOOLCHAIN_FILE= Users/filter/Desktop/adxe/cmake -DCMAKE_OSX_ARCHITECTURES=x86_x64

please kindly help me anyone to start the project..

**Comment by halx99** [comment]
The adxe not test on mac m1 yet, so we can't ensure support mac m1.

**Reply by ghost** [reply]
@halx99 thank you for your reply..  can you help me to understood this above ABI info failed error i think there is the issue because i successfully able to run version 3 on mac m1.. kindly need your support.

**Comment by halx99** [comment]
What's your cmake version? please try to update cmake to latest

**Reply by ghost** [reply]
@halx99 i am using this one.. @halx99 if you need any more info i am ready to give but please need your support to use "ADXE" 

% cmake - -version
cmake version 3.23.1

**Comment by halx99** [comment, marked as answer]
Try follow command in your project root directory:
```sh
cmake -S . -B build -GXcode -DCMAKE_TOOLCHAIN_FILE=$ADXE_ROOT/cmake/ios.mini.cmake -DCMAKE_OSX_ARCHITECTURES=x86_64
```

**Reply by ghost** [reply]
okay @halx99 i am trying this and update you here ASAP.. again Thanks you @halx99

**Comment by ghost** [comment]
@halx99 it's successfully & perfectly working..  working very very fine... @halx99 thank you for your extensive support.. i always remember your help.. now onward i will use that command to create any ios simulator project right.. thank you so much..

**Reply by halx99** [reply]
You are welcome
